springboot/src/main/java/ca/jrvs/apps/trading/controller/OrderController.java

```diff
@@ -0,0 +1,94 @@
+package ca.jrvs.apps.trading.controller;
+
+import ca.jrvs.apps.trading.model.domain.MarketOrder;
+import ca.jrvs.apps.trading.model.domain.SecurityOrder;
+import ca.jrvs.apps.trading.service.OrderService;
+import ca.jrvs.apps.trading.utils.ResponseExceptionUtils;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@Controller
+@RequestMapping("/order")
+@Api(value = "Order", produces = MediaType.APPLICATION_JSON_UTF8_VALUE)
```

Comment:
FYI, @API is a springfox annotation instead of springboot. it's appropriate to put it her.